src/api/session/client/Factory.cpp

```diff
@@ -114,18 +114,16 @@ auto Factory::BitcoinGenerationTransaction(
     const auto serializedVersion = boost::endian::little_int32_buf_t{version};
     const auto locktime = boost::endian::little_uint32_buf_t{0};
     const auto sequence = boost::endian::little_uint32_buf_t{0xffffffff};
-    const auto cb = [&] {
-        const auto bip34 =
-            opentxs::blockchain::bitcoin::block::internal::EncodeBip34(height);
-        auto output = space(bip34.size() + coinbase.size());
-        auto it = output.data();
-        std::memcpy(it, bip34.data(), bip34.size());
-        std::advance(it, bip34.size());
-        std::memcpy(it, coinbase.data(), coinbase.size());
-        output.resize(std::min<std::size_t>(output.size(), 100u));
-
-        return output;
-    }();
+
+    const auto bip34 =
+        opentxs::blockchain::bitcoin::block::internal::EncodeBip34(height);
+    auto cb = space(bip34.size() + coinbase.size());
```

Comment:
The name 'cb' appears to be a result of an ancient copy and paste accident. Perhaps 'bytes'? Unless you can think of something better, but 'cb' suggests a callback and that is wrong.